aries-site/src/pages/templates/lists.mdx

```diff
@@ -88,7 +88,7 @@ Examples are included for different screen sizes – Desktop, Laptop and Mobile.
 
 Lists should span the width of the container they are laid out in
 
-### Wrapping vs Truncation:
+### Wrapping vs truncation:
```

Comment:
@taysea 
Wrapping vs. truncation
[the Chicago Manual of Style seems to use vs. with the period, so I will go with that. We have it with and without in our guide, but for consistency, I would go with what the CMoS guide uses]